agent/xds/server.go

```diff
@@ -23,26 +23,30 @@ import (
 	"github.com/hashicorp/consul/agent/xds/xdscommon"
 )
 
-var StatsGauges = []prometheus.GaugeDefinition{
-	{
-		Name: []string{"xds", "server", "streams"},
-		Help: "Measures the number of active xDS streams handled by the server split by protocol version.",
-	},
-}
-
-var StatsCounters = []prometheus.CounterDefinition{
-	{
-		Name: []string{"xds", "server", "streamDrained"},
-		Help: "Counts the number of xDS streams that are drained when rebalancing the load between servers.",
-	},
-}
-
-var StatsSummaries = []prometheus.SummaryDefinition{
-	{
-		Name: []string{"xds", "server", "streamStart"},
-		Help: "Measures the time in milliseconds after an xDS stream is opened until xDS resources are first generated for the stream.",
-	},
-}
+var (
+	StatsGauges = []prometheus.GaugeDefinition{
+		{
+			Name: []string{"xds", "server", "streams"},
+			Help: "Measures the number of active xDS streams handled by the server split by protocol version.",
+		},
+		{
+			Name: []string{"xds", "server", "streamsUnauthenticated"},
```

Comment:
Does this read better as `unauthenticatedStreams`?

I thought it would be nicer to close to the related `streams` gauge when sorted alphabetically.